src/project/routers/auth.py

```diff
@@ -9,46 +9,59 @@
 from sqlalchemy.exc import IntegrityError, NoResultFound
 from sqlalchemy.orm import Session
 from starlette.responses import RedirectResponse, HTMLResponse
+from fastapi import Response
 
-from ..dependencies import get_db_session, login_user, get_user_from_session
+from ..dependencies import get_db_session, login_user, get_user_from_session, oauth
 from ..models.user import User
-from ...utils import hash_password, pwd_context
-from fastapi import Response
+from ..utils.errors.errors import UserDataError, OauthProviderError
+from ..utils.additional_func_auth import hash_password, pwd_context
+from ..utils.enums import OauthProvider
+from ..utils.errors.error_messages import OAUTH_NO_EMAIL, OAUTH_NO_USER_INFO, PASSWORD_REQUIRED, PASSWORDS_DO_NOT_MATCH, \
+    EMAIL_ALREADY_REGISTERED, GOOGLE_USER, USER_NOT_FOUND, INCORRECT_PASSWORD
 
 current_dir = os.path.dirname(os.path.realpath(__file__))
 template_dir = os.path.join(current_dir, "..", "templates")
 
 templates = Jinja2Templates(directory=template_dir)
-router = APIRouter(prefix="/auth")
 
+router = APIRouter()
 
 #Gets register template
 @router.get("/register", tags=["auth"], response_class=HTMLResponse)
-async def get_register_user(request: Request) -> HTMLResponse:
+async def register_user(request: Request) -> HTMLResponse:
     return templates.TemplateResponse("register.html", {"request": request})
 
 
 #Post view for user registration
 @router.post("/register", tags=["auth"], response_class=HTMLResponse)
-async def post_register_user(
+async def register_user(
```

Comment:
I remember I asked you to rename the methods, but it’s better to leave it as you did before. Rename them as get_register, post_register